aptos/api/aux-ts/src/client.ts

```diff
@@ -730,6 +749,8 @@ export class AuxClientError extends Error {
  * the AuxClient constructor.
  */
 export interface AuxClientOptions {
+  moduleAuthority: AptosAccount;
```

Comment:
Does the `moduleAuthority` ever change? If not maybe it should live on `AuxClient` directly similar to `moduleAddress`